x/nft/base/handler.go

```diff
@@ -0,0 +1,161 @@
+package base
+
+import (
+	"fmt"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/orm"
+	"github.com/iov-one/weave/x"
+	"github.com/iov-one/weave/x/nft"
+	"github.com/iov-one/weave/x/nft/blockchain"
+	"github.com/iov-one/weave/x/nft/ticker"
+	"github.com/iov-one/weave/x/nft/username"
+)
+
+const (
+	updateApprovalCost = 0
```

Comment:
this should be non-zero. maybe 100?
look at namecoin or cash for some reference values.
we should have the habit of making this semi-realistic for when we start enforcing fees and gas

Reply:
Will take a look then, wasn't sure how this should work